Tests/SlicePlotterPresenterTest.py

```diff
@@ -0,0 +1,83 @@
+from sliceplotter.sliceplotterpresenter import SlicePlotterPresenter,Axis
+from sliceplotter.SlicePlotter import SlicePlotter
+from sliceplotter.SlicePlotterView import SlicePlotterView
+from sliceplotter.command import Command
+from mainview import MainView
+from MainPresenter import MainPresenter
+import unittest
+import mock
+
+
+class SlicePlotterPresenterTest(unittest.TestCase):
+    def setUp(self):
+        # Setting up main tool presenter and view
+        self.main_view = mock.create_autospec(MainView)
+        self.main_presenter = mock.create_autospec(MainPresenter)
+        self.main_view.get_presenter = mock.Mock(return_value=self.main_presenter)
+        # Setting up view for slice plotter window
+        self.slice_plotter = mock.create_autospec(SlicePlotter)
+        self.slice_view = mock.create_autospec(SlicePlotterView)
+
+    def test_constructor_success(self):
+        self.slice_plotter_presenter = SlicePlotterPresenter(self.main_view, self.slice_view, self.slice_plotter)
+
+    def test_constructor_invalid_main_view_failure(self):
+        self.assertRaises(TypeError, SlicePlotterPresenter, self.slice_view, self.slice_view, self.slice_plotter)
```

Comment:
`ValueError` is probably better than `TypeError` unless you are really checking types (and you probably shouldn't be).